tests/Makefile

```diff
@@ -12,6 +12,12 @@ gherkin_fmt = ~/.local/bin/reformat-gherkin
 RM_DIR ?= true
 TEST_DIR ?= $(CURR_DIR)
 
+ifeq ($(ASAN), false)
+	NUM_THREADS=8
```

Comment:
The option named `J` is more clear.
And `J ?= 8`.

Reply:
fixed